mac-setup/build-app.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
```

Comment:
Would it be possible to use a consistent shebang: `#!/usr/bin/env bash`? I noticed the old one was already replaced in `mac-setup/complete-build.sh`